Jenkinsfile

```diff
@@ -465,17 +472,89 @@ EXIT /b 1
-            skipDefaultCheckout()
-          }
-          */
           steps {
             unstash name: 'docs'
             sh "test ! -z '${env.GIT_BRANCH}'"
             sh "test ! '${env.GIT_BRANCH}' = 'null'"
-            sshPublisher(publishers: [sshPublisherDesc(configName: 'OMSimulator-doc', transfers: [sshTransfer(execCommand: "rm -rf .tmp/${env.GIT_BRANCH}"), sshTransfer(execCommand: "test ! -z '${env.GIT_BRANCH}' && rm -rf '/var/www/doc/OMSimulator/${env.GIT_BRANCH}' && mkdir -p `dirname '/var/www/doc/OMSimulator/.tmp/${env.GIT_BRANCH}'` && mv '/var/www/doc/OMSimulator/.tmp/${env.GIT_BRANCH}' '/var/www/doc/OMSimulator/${env.GIT_BRANCH}'", remoteDirectory: ".tmp/${env.GIT_BRANCH}", removePrefix: "install/linux/doc", sourceFiles: 'install/linux/doc/**')])])
+            sshPublisher (
+              publishers: [
+                sshPublisherDesc(
+                  configName: 'OMSimulator-doc',
+                  transfers: [
+                    sshTransfer(execCommand: "rm -rf .tmp/${env.GIT_BRANCH}"),
+                    sshTransfer(
+                      execCommand: "test ! -z '${env.GIT_BRANCH}' && rm -rf '/var/www/doc/OMSimulator/${env.GIT_BRANCH}' && mkdir -p `dirname '/var/www/doc/OMSimulator/.tmp/${env.GIT_BRANCH}'` && mv '/var/www/doc/OMSimulator/.tmp/${env.GIT_BRANCH}' '/var/www/doc/OMSimulator/${env.GIT_BRANCH}'",
+                      remoteDirectory: ".tmp/${env.GIT_BRANCH}",
+                      removePrefix: "install/linux/doc",
+                      sourceFiles: 'install/linux/doc/**')
+                  ]
+                )
+              ]
+            )
+          }
+        }
+
+        stage('upload-artifacts') {
+          when {
+            allOf {
+              not {
+                changeRequest()
+              }
+              anyOf {
+                buildingTag()
+                anyOf {
+                  branch 'master'
+                  branch 'jenkins' // For testing purposes
+                  branch 'maintenance/**'
+                }
+              }
+            }
+            beforeAgent true
+          }
+          agent {
+            label 'linux'
+          }
+          steps {
+            unstash name: 'amd64-zip'         // includes: "OMSimulator-linux-amd64-*.tar.gz"
+            unstash name: 'arm32-zip'         // includes: "OMSimulator-linux-arm32-*.tar.gz"
+            unstash name: 'i389-zip'          // includes: "OMSimulator-linux-i386-*.tar.gz"
+            unstash name: 'mingw32-zip'       // includes: "OMSimulator-mingw32*.zip"
+            unstash name: 'mingw64-zip'       // includes: "OMSimulator-mingw64*.zip"
+            unstash name: 'win64-zip'         // includes: "OMSimulator-win64*.zip"
+            unstash name: 'osx-zip'           // includes: "OMSimulator-osx*.zip"
+
+            sh "ls *.zip *.tar.gz"
+
+            sshPublisher (
+              publishers: [
+                sshPublisherDesc(
+                  configName: 'OMSimulator-doc',
```

Comment:
This needs to be "OMSimulator" instead "OMSimulator-doc".

Reply:
Changed it.